CONTRIBUTING.md

```diff
@@ -1,10 +0,0 @@
-By submitting a pull request, you represent that you have the right to license
```

Comment:
suggestion (non-blocking): In the event someone was to land on this repository first, can we add some wording that directs the reader to the swiftlang contribution documentation?